nautobot/ui/README.md

```diff
@@ -0,0 +1,21 @@
+# Nautobot v2 UI Prototype
+
+## Running it
+
+Install and compile dependencies:
+
+```no-highlight
+nautobot-server build
```

Comment:
```suggestion
nautobot-server build_ui
```

Do we need to include `--npm-install` on the first invocation of this command?

Reply:
Yes. Thanks.